permissions/plugin-audit-log.yml

```diff
@@ -0,0 +1,10 @@
+---
+name: "audit-log"
+github: "jenkinsci/audit-log-plugin"
+paths:
+  - "org/jenkins-ci/plugins/audit-log"
```

Comment:
Note: generally, for new plugins, the `groupId` seen more has become `io.jenkins.plugins`. You might want to change to this before your actual first release (?)

Reply:
Thanks for the suggestion. Filed as [JENKINS-55451](https://issues.jenkins-ci.org/browse/JENKINS-55451).

Reply:
thanks @batmat, wasn't sure if it was suppose to be `org` or `io`. I saw you had commented in an earlier PR that it should be `io`, but was looking through the other plugins in the permissions directory and saw alot of `org` ones too. Will switch it to `io` in abit.

Reply:
there's no strict rule really. `io.jenkins.plugins` just makes more sense for brand new plugins, given 1) we now own https://jenkins.io domain and 2) Jenkins is now much more than just a "CI" platform hence why also we moved away from using that suffix in places where we could.

Reply:
ah.. i see. interesting.

Reply:
Don't forget to update pom.xml as well in the other repo.

Reply:
@batmat is the `io` change specific to all instances of `org.jenkins-ci.plugins` ? or applicable more broadly, e.g. `org.jenkins-ci.tools` and `org.jenkins-ci.xxxx` ?

Reply:
never mind, just re-read your comment that it's for brand new plugins